float64  speed
float64  angle_forward
float64  angle_rear
---
bool retval